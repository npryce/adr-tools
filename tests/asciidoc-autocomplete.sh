_adr_autocomplete adr
_adr_autocomplete adr li
_adr_autocomplete adr generate
_adr_autocomplete adr generate to
_adr_autocomplete adr ierhir
