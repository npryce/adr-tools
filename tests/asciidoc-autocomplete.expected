_adr_autocomplete adr
config generate help init link list new upgrade-repository
_adr_autocomplete adr li
link list
_adr_autocomplete adr generate
graph toc
_adr_autocomplete adr generate to
toc
_adr_autocomplete adr ierhir

